packages/cisco_asa/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.37.1"
+  changes:
+    - description: "Customer is having poor experience when logs are missing message_id field."
```

```suggestion
    - description: "Remove erroneous check for empty message ID in kv processor."
```
We don't need to mention a customer.